src/data/template/Microsoft.Authorization/policyAssignments.jq

```diff
@@ -0,0 +1 @@
+del(.ResourceId, .Properties.Metadata, .ResourceGroupName, .SubscriptionId)
```

Comment:
Appreciate this probably works, but shouldn't these keys be in camelCase?

Reply:
JQ is case sensitive and Built-in PowerShell class has these names.

Reply:
![image](https://user-images.githubusercontent.com/14359777/111615203-5c877900-87d8-11eb-9d37-06fb1ae9d02a.png)